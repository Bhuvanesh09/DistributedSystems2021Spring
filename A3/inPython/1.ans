322398
322398
322398
322292
322292
320233
320233
317230
314822
314822
