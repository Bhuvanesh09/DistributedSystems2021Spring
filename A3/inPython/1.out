314611
314611
314611
314611
314611
314611
314611
314611
314611
314611
