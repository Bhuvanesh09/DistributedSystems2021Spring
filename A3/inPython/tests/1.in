add_graph g1 100
add_edge g1 59 100 2747
add_edge g1 91 44 4262
add_edge g1 71 66 5700
add_edge g1 60 90 6832
add_edge g1 13 21 2445
add_edge g1 85 58 3849
add_edge g1 51 41 7323
add_edge g1 99 56 9495
add_edge g1 83 93 6177
add_edge g1 89 26 7671
add_edge g1 82 82 6462
add_edge g1 57 40 9546
add_edge g1 35 83 4902
add_edge g1 82 65 6603
add_edge g1 77 86 2945
add_edge g1 26 73 4452
add_edge g1 30 29 8753
add_edge g1 68 52 3322
add_edge g1 81 94 937
add_edge g1 42 7 4010
add_edge g1 44 76 5398
add_edge g1 79 60 4219
add_edge g1 80 100 5991
add_edge g1 44 69 1261
add_edge g1 15 68 1740
add_edge g1 57 46 9763
add_edge g1 50 32 807
add_edge g1 80 66 5890
add_edge g1 62 89 2309
add_edge g1 45 32 452
add_edge g1 32 55 769
add_edge g1 32 83 467
add_edge g1 53 44 5427
add_edge g1 87 35 4918
add_edge g1 84 4 1209
add_edge g1 52 43 3387
add_edge g1 7 35 4992
add_edge g1 77 48 2820
add_edge g1 20 85 2
add_edge g1 33 89 7736
add_edge g1 66 50 9360
add_edge g1 14 24 7626
add_edge g1 32 33 806
add_edge g1 65 68 6347
add_edge g1 72 90 2216
add_edge g1 84 2 975
add_edge g1 77 85 2761
add_edge g1 23 15 144
add_edge g1 23 31 396
add_edge g1 39 11 3201
add_edge g1 72 93 1759
add_edge g1 85 37 3678
add_edge g1 5 60 3612
add_edge g1 65 26 8359
add_edge g1 48 40 4006
add_edge g1 65 30 8646
add_edge g1 78 52 3299
add_edge g1 59 36 2737
add_edge g1 49 4 6800
add_edge g1 64 66 1492
add_edge g1 24 73 4468
add_edge g1 59 25 2718
add_edge g1 27 76 8979
add_edge g1 3 23 432
add_edge g1 1 92 8557
add_edge g1 57 97 9864
add_edge g1 35 71 4738
add_edge g1 65 38 8617
add_edge g1 26 73 4385
add_edge g1 3 13 5092
add_edge g1 99 17 9390
add_edge g1 25 26 3596
add_edge g1 53 77 2950
add_edge g1 94 60 6864
add_edge g1 88 34 9179
add_edge g1 82 98 6536
add_edge g1 73 76 4542
add_edge g1 22 84 91
add_edge g1 66 29 8717
add_edge g1 2 50 8195
add_edge g1 96 9 7270
add_edge g1 65 2 8183
add_edge g1 25 42 3545
add_edge g1 46 49 6623
add_edge g1 59 34 2741
add_edge g1 84 60 961
add_edge g1 29 86 8666
add_edge g1 18 69 1286
add_edge g1 51 90 9046
add_edge g1 84 50 1186
add_edge g1 58 70 6128
add_edge g1 76 23 185
add_edge g1 89 57 7909
add_edge g1 18 55 8314
add_edge g1 30 58 6138
add_edge g1 34 28 4051
add_edge g1 1 41 7336
add_edge g1 11 69 1351
add_edge g1 27 58 6019
add_edge g1 62 19 2301
add_edge g1 52 65 3403
add_edge g1 93 88 6188
add_edge g1 70 83 6446
add_edge g1 86 21 2551
add_edge g1 7 41 7466
add_edge g1 75 82 1663
add_edge g1 43 66 8383
add_edge g1 44 57 5513
add_edge g1 92 20 35
add_edge g1 55 10 8242
add_edge g1 35 20 11
add_edge g1 19 15 1716
add_edge g1 17 62 2365
add_edge g1 1 57 9572
add_edge g1 44 79 4217
add_edge g1 17 31 9664
add_edge g1 41 6 4316
add_edge g1 78 56 5777
add_edge g1 99 48 3887
add_edge g1 85 79 3708
add_edge g1 100 32 901
add_edge g1 75 82 1650
add_edge g1 23 90 339
add_edge g1 37 78 4173
add_edge g1 86 52 3444
add_edge g1 61 69 1286
add_edge g1 32 89 632
add_edge g1 46 90 9217
add_edge g1 66 84 988
add_edge g1 16 38 9176
add_edge g1 22 18 100
add_edge g1 8 12 8135
add_edge g1 32 36 635
add_edge g1 25 39 3123
add_edge g1 72 68 1908
add_edge g1 12 96 7081
add_edge g1 34 74 9090
add_edge g1 28 78 4046
add_edge g1 21 45 2493
add_edge g1 53 64 1492
add_edge g1 87 53 8158
add_edge g1 44 7 5401
add_edge g1 27 9 9200
add_edge g1 7 70 9686
add_edge g1 86 89 7721
add_edge g1 73 70 4662
add_edge g1 3 88 9194
add_edge g1 63 38 8492
add_edge g1 37 79 4147
add_edge g1 94 89 7749
add_edge g1 49 80 5789
add_edge g1 63 6 4372
add_edge g1 53 29 8795
add_edge g1 43 89 7704
add_edge g1 95 12 8880
add_edge g1 34 28 4057
add_edge g1 4 44 5555
add_edge g1 96 62 2304
add_edge g1 95 98 8359
add_edge g1 18 75 1693
add_edge g1 8 18 8058
add_edge g1 58 11 6120
add_edge g1 65 63 8450
add_edge g1 12 8 8098
add_edge g1 82 26 6579
add_edge g1 40 4 9567
add_edge g1 56 89 7816
add_edge g1 24 45 5596
add_edge g1 43 92 8369
add_edge g1 62 35 2261
add_edge g1 53 100 9842
add_edge g1 25 83 3565
add_edge g1 87 41 7279
add_edge g1 10 96 6935
add_edge g1 33 3 9722
add_edge g1 21 70 2575
add_edge g1 13 18 5059
add_edge g1 67 77 3004
add_edge g1 98 39 3167
add_edge g1 14 90 7653
add_edge g1 37 19 4182
add_edge g1 36 98 8157
add_edge g1 54 63 2641
add_edge g1 75 37 1645
add_edge g1 97 90 9218
add_edge g1 81 91 907
add_edge g1 73 85 3833
add_edge g1 55 95 8249
add_edge g1 28 53 4101
add_edge g1 72 47 2123
add_edge g1 57 9 9765
get_mst g1
add_edge g1 4 69 1392
get_mst g1
add_edge g1 95 90 8853
get_mst g1
add_edge g1 87 14 7520
get_mst g1
add_edge g1 26 61 6408
get_mst g1
add_edge g1 32 21 434
get_mst g1
add_edge g1 43 44 5143
get_mst g1
add_edge g1 98 23 164
get_mst g1
add_edge g1 21 87 2510
get_mst g1
add_edge g1 5 36 3644
get_mst g1
