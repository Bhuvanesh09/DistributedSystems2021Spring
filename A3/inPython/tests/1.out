Hello, Client! You are connected from 127.0.0.1:57842.
